docs/manual/java/gettingstarted/GettingStartedMaven.md

```diff
@@ -9,8 +9,14 @@ Follow these instructions to create and run your first project:
 
 ## Generate a project with the Lagom archetype
 
-To create a project, follow these steps:  
+To create a project, open a command window and follow these steps:
 
+1. Create a new directory for your build. For example:
+   ```
+   mkdir lagom
+   ```
+
```

Comment:
Logically, 2 should come before 1 :-). I've changed the wording, see what you think. I should change the pages on the website to be similar.